sources/api/apiserver/src/server/exec/child.rs

```diff
@@ -241,6 +241,16 @@ impl ChildHandles {
     }
 }
 
+impl Drop for ChildHandles {
+    /// Ensures the RawFd is properly closed when ChildHandles is dropped.
+    fn drop(&mut self) {
+        match close(self.pty_fd) {
+            Ok(_) => debug!("Closed file descriptor: {}", self.pty_fd),
+            Err(e) => warn!("Failed to close file descriptor {}: {}", self.pty_fd, e),
+        }
+    }
```

Comment:
Not sure we want to do much or any logging in a `Drop` impl:

```suggestion
    fn drop(&mut self) {
        let _ = close(self.pty_fd);
    }
```

Reply:
Fair 👍 I saw we did a warning in [Termina's Dropl](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/develop/sources/api/apiclient/src/exec/terminal.rs#L93) but I agree it doesn't add much value.